src/test/java/com/google/api/generator/gapic/composer/GrpcServiceCallableFactoryClassComposerTest.java

```diff
@@ -35,6 +38,10 @@
 public class GrpcServiceCallableFactoryClassComposerTest {
   private ServiceDescriptor echoService;
   private FileDescriptor echoFileDescriptor;
+  private static final String GOLDENFILES_DIRECTORY =
+      "src/test/java/com/google/api/generator/gapic/composer/goldens/";
+  private static final String GOLDENFILES_NAME =
```

Comment:
Would prefer for each filename to be local to each test method, since this is a per-instance type of data.

Reply:
moved the golden file name as a local string instead of global constant.